nevergrad/benchmark/experiments.py

```diff
@@ -750,7 +761,12 @@ def powersystems(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
     algos += ["ProgOptimizer3", "ProgOptimizer5", "ProgOptimizer9", "ProgOptimizer13",
               "ProgDOptimizer3", "ProgDOptimizer5", "ProgDOptimizer9", "ProgDOptimizer13",
               "OptimisticNoisyOnePlusOne", "OptimisticDiscreteOnePlusOne"]
-    for budget in [1600, 3200, 6400, 12800]:
+    RS = np.random.RandomState(next(seedg))
```

Comment:
don't use capital letters for non global variables (and preferably, don't use global variables anyway^^)

Reply:
Fixed.